src/no/runsafe/UserControl/database/PlayerDatabase.java

```diff
@@ -78,7 +114,7 @@ public void logPlayerInfo(IPlayer player)
 	{
 		console.debugFine("Updating player_db with login time");
 		database.update(
-			"INSERT INTO player_db (`uuid`,`name`,`joined`,`login`,`ip`) VALUES (?,?,NOW(),NOW(),INET_ATON(?))" +
+			"INSERT INTO player_db (`uuid`, `name`, `joined`,`login`,`ip`) VALUES (?,?,NOW(),NOW(),INET_ATON(?))" +

```

Comment:
I try to avoid changing queries in previous revisions, even though this commit is fine, it creates noise in the file history